### DISCUSSION TITLE
cleaning up output folder path used to execute .\Run.bat installation

### DISCUSSION THREAD

**Discussion by myusrn** [LSPosed/MagiskOnWSALocal#184] [Q&A] [answered]
After building on wsl ubuntu i moved the ./output content to windows host $env:userprofile\Downloads\WSA-with-magisk-stable-MindTheGapps_2207.40000.8.0_x64_Release-Nightly and ran the .\Run.bat [ -> Install.ps1 ] installation process from there.  

Now when i try and delete the $env:userprofile\Downloads\WSA-with-magisk-stable-MindTheGapps_2207.40000.8.0_x64_Release-Nightly explorer says it can't as some files are in use.

q1. did the installation process setup my wsa with magisk and mindthegapps configuration using the folder i launched .\Run.bat from as the install directory?

q2. if answer to q1 is yes do i have a way move moving that install directory after the fact without losing my wsa configuration or do i have to uninstall and then reinstall, loosing prior configuration and app installs, with that output installer directly located in the place i want it to be?

**Comment by Howard20181** [comment, marked as answer]
A1: Yes.

A2: Stop WSA using WSA setting then close WSA setting, move that folder to another places, rerun `Run.bat`.

**Reply by myusrn** [reply]
Thanks, worked easily and quickly allowing me to move from $env:userprofile\Downloads\<wsa magisk build ouput folder name> to c:\wsaMagiskGapps folder instead. I expect i now also have answer on how i upgrade when new wsa drop and/or wsa with magisk and mindthegapps installation comes up and retain existing settings.

**Comment by myusrn** [comment]
@Howard20181 

If my win11 22h2 setup is configured to be using insider dev channel, e.g. currently 25217.1010, then when i build the wsa with magisk and mindthegapps x64 output do i need to be selecting the matching release type when ./scripts/run.sh asks me?

If the answer is yes should i expect to have to rebuild the wsa with magisk and mindthegapps x64 output, and execute the ./Run.bat update process against my existing install with that new build output, every time there is a new win11 22h2 insider dev channel update?

![image](https://user-images.githubusercontent.com/3043446/196510426-ac1fc40f-1247-4626-b64b-a814861fcbf0.png)

**Reply by Howard20181** [reply]
No, you don't. The two preview channels are separate.
